### PR TITLE
Make testenv playbooks easier to run from WSL

### PR DESCRIPTION
**Pull request by sodar**
- Added `vars_prompt` for required parameters to `vmware-{deploy,destroy}-testenv.yml`
- `testenv-conf.yaml` is generated to temporary file if path was not provided
- Moved vCenter credentials to vault
- Updated related Jenkinsfiles

To do:

- [x] ~~Run manually~~
- [x] ~~Run `deploy-dev-env` job~~
- [x] ~~Run `deploy-demo-env` job~~
- [x] ~~Run `destroy-env` job~~

### CODE REVIEW

**Comment by sodar** [comment]
Started `deploy-dev-env`: http://10.84.12.26:8080/job/WinContrail-infra/job/deploy-dev-env/64/console

**Comment by sodar** [comment]
`destroy-env` works: http://10.84.12.26:8080/job/WinContrail-infra/job/destroy-env/95/console

File: ansible/group_vars/controller/vars [hunk 1]
```diff
@@ -1,3 +1,3 @@
 ansible_user: root
 ansible_ssh_pass: contrail123
-ansible_ssh_extra_args: "-o UserKnownHostsFile=/dev/null"
+ansible_ssh_extra_args: "-o StrictHostKeyChecking=no"
```
Comment (jablonskim) [inline ansible/group_vars/controller/vars]: I think that we need also `UserKnownHostsFile` here

**Comment by sodar** [comment]
recheck no bug

**Pull request merged**